### PR TITLE
enhancement/Issue 1171: Include protocol with logged urls in terminal output

### PR DESCRIPTION
**Pull request by DevLab2425**
## Related Issue

https://github.com/ProjectEvergreen/greenwood/issues/1171

## Summary of Changes

- Include the `http://` protocol on console logs used during server startup.

### CODE REVIEW

**Review by thescientist13 — APPROVED** [review]
Nice!

One favor, if you could re-target this PR against the **release/0.29.0** branch, I can get this out in the next alpha release for the next minor release (which should be coming out in a couple weeks! 🤞 )

**Comment by DevLab2425** [comment]
> if you could re-target this PR against the release/0.29.0 branch

@thescientist13 I've updated the target here. When it's ready, feel free to take it from here.

**Comment by thescientist13** [comment]
Thanks @DevLab2425 !  Will leave it open for a couple days for other folks to review and then I'll take it from there.

File: packages/plugin-renderer-puppeteer/src/plugins/server.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ class PuppeteerServer extends ServerInterface {
       const { port } = this.compilation.config.devServer;
 
       (await getDevServer(this.compilation)).listen(port, async () => {
-        console.info(`Started puppeteer prerender server at localhost:${port}`);
+        console.info(`Started puppeteer prerender server at http://localhost:${port}`);
```
Comment (thescientist13) [inline packages/plugin-renderer-puppeteer/src/plugins/server.js:15]: This one probably won't ever get used by the user (just run at build time) but can't hurt!

**Pull request merged**